src/ucp/rndv/rndv.c

```diff
@@ -508,7 +507,8 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_get_zcopy, (self),
         length = ucp_mtu - remaining;
     } else {
         chunk = ucs_align_up((size_t)(rndv_req->send.length /
-                                      rndv_req->send.rndv_get.lanes_count
+                                      ucs_popcount(rndv_req->send.
+                                                   rndv_get.lanes_map_all)
```

Comment:
popcount can be slow
lets keep lane_count, remove lanes_map_all, and do UCS_MASK(lane_count) when needed

Reply:
not sure that is will work, since `lanes_map_all` contains only lane idx selected in `ucp_rndv_req_init_get_zcopy_lane_map()`

Reply:
yes, that's right
so we must keep both, cannot add popcount to datapath

Reply:
or, maybe unite lanes_map_avail with am_bw_index?

Reply:
> or, maybe unite lanes_map_avail with am_bw_index?

yes, good idea
will do

Reply:
done